tox.ini

```diff
@@ -1,4 +1,4 @@
 [flake8]
-ignore = F401,E501,E402,E401
+ignore = F401,E501,E402,E401, E305, E302, N812
```

Comment:
1. You should keep the spacing consistent (nit pick).

2. Is there really a need to call the variable `VERSION`?  If we didn't do that, we could remove N812

3. This file suggests commenting each exception: http://flake8.pycqa.org/en/latest/user/configuration.html

4. Gee, E302 is silly!  :-)  We have to ignore that one.

5. We don't want to get rid of "E305 expected 2 blank lines after class or function definition, found 1", do we?  Why not just insert a blank line?